### ISSUE TITLE
Make App Store icon work with VoiceOver

### ISSUE THREAD

**Issue by mi-wood** [RefugeRestrooms/refugerestrooms-ios#149]
@DeeDeeG : 

> The App Store download button isn't focusable with VoiceOver, so it would be nice to make that focusable. Not sure how at the moment, but should be doable.

**Comment by hkellaway** [comment]
@mi-wood from that description, it sounds like it's actually a web issue - the button on the website isn't configured to be read by the iOS screen reader (VoiceOver). I don't know a thing about web + VoiceOver, only a bit about native apps - maybe an article like this has some insight  https://www.deque.com/blog/accessible-aria-buttons/

**Comment by mi-wood** [comment]
oops! I misread this. moving it over
